### PR TITLE
[dev] No PBI - Fixes Sectional Nav Indicator

### PR DESCRIPTION
**Pull request by cbrewer1689**

### CODE REVIEW

**Comment by groberts314** [comment]
We've got a failed unit test:

https://dev.azure.com/saddlebackchurch/Church%20Management/_build/results?buildId=40568&view=logs&j=fd490c07-0b22-5182-fac9-6d67fe1e939b&t=ec339c2b-c439-539d-f401-d5569bbc95f9&l=585

```
Summary of all failing tests
FAIL src/navigation/sectionalTabs/__test__/sectionalTabs.test.js
  ● <SectionalTabs /> › renders 5 tabs as hidden when the container width is lower than the tabs total width

    expect(received).toEqual(expected) // deep equality

    - Expected  - 6
    + Received  + 1

    - Array [
    -   "Example Tab 1",
    -   "Example Tab 2",
    -   "Example Tab 3",
    -   "Example Tab 4",
    - ]
    + Array []

      50 |         wrapper.update();
      51 |         const visibleTabs = wrapper.find('SectionalTab').find('h4').map((tab) => tab.prop('children'));
    > 52 |         expect(visibleTabs).toEqual([
         |                             ^
      53 |             'Example Tab 1',
      54 |             'Example Tab 2',
      55 |             'Example Tab 3',

      at Object.<anonymous> (src/navigation/sectionalTabs/__test__/sectionalTabs.test.js:52:29)


Test Suites: 1 failed, 24 passed, 25 total
Tests:       1 failed, 1 todo, 252 passed, 254 total
Snapshots:   0 total
```

File: src/navigation/sectionalTabs/sectionalTabs.jsx [hunk 1]
```diff
@@ -83,17 +87,22 @@ const defaultProps = {
     selectedTabKey: undefined,
     withContent: false,
 };
-
-const useStyles = (theme) => {
-    const borderColorContrastPrimary = get(theme, 'palette.border.contrastPrimary');
+const styles = (theme) => {
```
Comment (groberts314) [inline src/navigation/sectionalTabs/sectionalTabs.jsx:90]: NAB: Would love to keep the blank line in between 89 and 90 here if we can.

Comment (cbrewer1689) [reply src/navigation/sectionalTabs/sectionalTabs.jsx]: Fixed.

**Comment by cbrewer1689** [comment]
Okay, the unit test failure has been addressed. Thanks for the patience

File: src/navigation/sectionalTabs/sectionalTabs.jsx [hunk 2]
```diff
@@ -14,14 +14,17 @@ import {
...
     BEM_NAVIGATION_SECTIONAL_TABS,
     BEM_NAVIGATION_TAB_ROOT_CLASS,
 } from '../../global/constants';
+import {
+    PADDING_X,
+} from './constants';
+import SectionalTab from './sectionalTab';
+import SectionalTabContent from './sectionalTabContent';
+import withStyles from '../../styles/withStyles';
+import DropdownButton from '../../inputs/dropdownButton';
```
Comment (groberts314) [inline src/navigation/sectionalTabs/sectionalTabs.jsx:27]: NAB: Should this be sorted above Line 24 `import SectionalTab from './sectionalTab';` for some alphabetical ordering?

**Review by groberts314 — APPROVED** [review]
Re-reviewed again after latest changes and still lookin' pretty darn good to me.  I'm excited to have this issue fixed properly too!

File: src/navigation/sectionalTabs/sectionalTabs.jsx (hunk 2 above)
Comment (cbrewer1689) [reply src/navigation/sectionalTabs/sectionalTabs.jsx]: Fixed.

**Pull request merged**